plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -102,6 +103,29 @@ export type ApplyConditionsResponse = {
   items: ApplyConditionsResponseEntry[];
 };
 
+/**
+ * Serialized permission rules, with the paramsSchema
+ * converted from a ZodSchema to a JsonSchema.
+ *
+ * @public
+ */
+export type MetaDataResponseSerializedRule = {
+  name: string;
+  description: string;
+  resourceType: string;
+  paramsSchema?: JsonSchema7Type;
+};
+
+/**
+ * Response type for the .metadata endpoint.
+ *
+ * @public
+ */
+export type MetaDataResponse = {
```

Comment:
Should this be `PermissionsMetadataResponse` instead?

Reply:
I had an internal battle with this one, I agree it should be prefixed however there's another exported type in this file called `ApplyConditionsResponse` so I went with consistency. What do you think? Go for the prefix or match it up with the other one.

Reply:
Alright, let's keep it short. Could you switch it to `MetadataResponse` though? That's the casing we use everywhere else.